.github/workflows/cron-jobs.yaml

```diff
@@ -1,12 +1,13 @@
 name: Security and quality nightly scan
 
 on:
-  schedule:
-    - cron: '0 10 * * *'
+  pull_request:
+  # schedule:
+  #   - cron: '0 10 * * *'
```

Comment:
obviously revert this

Reply:
There's a new problem now: https://github.com/canonical/k8s-snap/actions/runs/11019106186/job/30604131753?pr=694#step:7:7502